mlir/utils/jenkins/Jenkinsfile.miopen-test

```diff
@@ -106,15 +103,21 @@ mlir/utils/jenkins/miopen-tests/**\
                         }
                     }
                     stage("Test MIOpen config") {
+                        environment {
+                            // Make libMIOpen.so become accessible to the test driver
+                            LD_LIBRARY_PATH="$LD_LIBRARY_PATH:${WORKSPACE}/MIOpen/build/lib"
```

Comment:
So this is the real fix?

Jenkinsfile should be updated as well?

Reply:
You mean the one big jenkinsfile? I'd expect the migration to that in another PR because this is technically to get the baseline correctly. Then before the transition I'd file another one to make sure the one big Jenkinsfile continue to pass.

Reply:
Sounds good to me.